src/Apps/Order/Utils/getErrorDialogCopy.tsx

```diff
@@ -4,6 +4,7 @@ import React from "react"
 const SUPPORT_EMAIL = "orders@artsy.net"
 
 export enum ErrorDialogs {
+  DestinationCouldNotBeGeocoded = "destination_could_not_be_geocoded",
   CurrencyNotSupported = "currency_not_supported",
 }
```

Comment:
non-blocking: Similar to my comment here https://github.com/artsy/force/pull/12803#discussion_r1310658734, we could make these values a graphql enum for free always up-to-date types.